src/cli/study-backrefs.js

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env node
+const requireFromWorkingDirectory = require('../lib/util').requireFromWorkingDirectory;
+const {canonicalizeUrl, canonicalizesTo} = require('../../builds/canonicalize-url');
+const fs = require("fs");
+
+const matchSpecUrl = url => url.match(/spec.whatwg.org/) || url.match(/www.w3.org\/TR\/[a-z0-9]/) || (url.match(/.github.io/) && ! url.match(/w3c.github.io\/test-results\//));
```

Comment:
What about `.csswg.org`, `.fxtf.org` and `.css-houdini.org` URLs?
Also, did you exclude Khronos Group specs on purpose?

Reply:
I didn't (or at least I don't recall picking that subset on purpose) - I guess the list of bad links will grow quite a bit more…